setup_symlinks.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+## Log and model folders can grow very large. This script creates symlinks to outsource them.
+## It requires a target directory as single parameter, e.g:
+##
+## bash setup_symlinks.sh $HOME/experiments/my-project
+##
+## Note, that training models with a different name than "default" may result in a different
+## save_dir (see config/train.yaml) which needs to be symlinked manually, if outsourcing them
+## is intended.
+
+TARGET_DIR=$1
+[[ -z "$TARGET_DIR" ]] && { echo "Error: No target directory was provided"; exit 1; }
+DIRS=( "logs/experiments" "logs/evaluations" "logs/debugs" "models/default" "models/conll2003" )
```

Comment:
Does it make sense to add `models/conll2003` here? And why `models/default/`? Can't it just be `models`, as the respective model dirs in `models/` are created during training anyway?

Reply:
Yeah, I was thinking about the same. But I wrote it like the current version to have an empty `models` folder in the repo when you initially create it from the template to talk about it in the [Project Structure section](https://github.com/ChristophAlt/pytorch-ie-hydra-template#project-structure) and I thought I may be confusing if this is not there initially. But you say, this may be fine when considering the decreased complexity, am I right?

Reply:
implemented in 79cb1eb